tests/unit/rankers/test_algorithms.py

```diff
@@ -288,42 +293,50 @@ def test_personalize_topic_limit(self):
                                for t in full_topic_profile.curator_topics
                                if t.curator_topic_label in input_topics]
 
-        for test_limit in [1, 3, 5]:
-            output_configs = personalize_topic_slates(input_configs, full_topic_profile, test_limit)
+        for test_limit, topic_ranker in zip([1, 3], [top1_topics, top3_topics]):
+            output_configs = topic_ranker(input_configs, full_topic_profile)
```

Comment:
I'll enter this as further evidence in support of "if it becomes more than 2, let's rethink how we do this"